### PR TITLE
Add inline-start border to filters on desktop

### LINKED ISSUE #729: Border style in Filters sidebar
## Description
The Filters sidebar does not have a border on the left side.

CleanShot 2022-01-27 at 11 14 38@2x

## Expectation
It should have a `1px` border of `dark-charcoal-20` color.

CleanShot 2022-01-27 at 11 14 52@2x

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #729 by @panchovm 

## Description

This PR adds a border to the left (in LTR, and right in RTL) side of the filter sidebar on desktop screen. To be honest,  with my settings for scrollbar visibility, it is not easy to see:

Screen Shot 2022-01-28 at 6 37 07 AM

## Testing Instructions

Run the app, and open the filters sidebar. You should see a border between the sidebar and the main content. 
## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by obulat** [comment]
Screenshot after changing the scrollbar properties and setting the locale direction to RTL
Screen Shot 2022-01-28 at 6 50 12 AM

File: src/components/VFilters/VSearchGridFilter.vue [hunk 1]
```diff
@@ -107,8 +102,18 @@ export default {
...
-      isMinScreenMd,
     }
   },
 }
 </script>
+<style scoped>
+.filters {
+  border-inline-start: 1px solid transparent;
+}
+@media (min-width: 768px) {
```
Comment (dhruvkb) [inline src/components/VFilters/VSearchGridFilter.vue:113]: Tailwind's `@screen` might be better here.
```suggestion
@screen md {
```

**Review by dhruvkb — APPROVED** [review]
Great implementation. Minor code style nitpick. Approving to unblock.

**Comment by panchovm** [comment]
It looks great ✨ 

> To be honest, with my settings for scrollbar visibility, it is not easy to see

Same to me when scrolling, but the section division is visible once the scroll bar disappears. Many people have different settings where the scrolling effect looks unalike.

**Comment by obulat** [comment]
> Great implementation. Minor code style nitpick. Approving to unblock.

Thank you, @dhruvkb , I wasn't aware of the @screen option!

**Review by panchovm — APPROVED** [review]
Great work ✨

**Pull request merged**